pulsar/connection/connection.go

```diff
@@ -75,11 +77,24 @@ func (*Factory) NewManager(settings map[string]interface{}) (connection.Manager,
 		}
 	}
 
+	customLogger := plogger.NewLoggerWithLogrus(logrus.StandardLogger())
+
+	// set debug log level if FE engine configured for 'DEBUG' else ERROR level
+
+	fll := os.Getenv("FLOGO_LOG_LEVEL")
```

Comment:
Instead of hardcoding FLOGO_LOG_LEVEL, you should use **log.EnvKeyLogLevel**.

Reply:
Made corresponding changes into commit f71f7a6.